examples/basics/datasets.ipynb

```diff
@@ -1,18 +1,16 @@
 {
```

Comment:
<ul><li>"Attaching a dataset to a project will add all data rows in the dataset to the project (and add them to the queue)"</li></ul><p>Can we remove this text?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>